From 5efd215f78b594c75141a5d4924388331bdf4f00 Mon Sep 17 00:00:00 2001
From: Luc Schrijvers <begasus@gmail.com>
Date: Fri, 18 Apr 2025 16:24:35 +0200
Subject: Revert requirement for Qt version from 6.8.0

This works for now, but will fail in the future

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 533eccd..21abdae 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -14,7 +14,7 @@ project(tokodon VERSION ${RELEASE_SERVICE_VERSION})
 option(SAILFISHOS "Build for sailfishOS" OFF)
 
 set(KF_MIN_VERSION "6.12.0")
-set(QT_MIN_VERSION "6.8.0")
+set(QT_MIN_VERSION "6.7.0")
 
 include(FeatureSummary)
 
-- 
2.50.1


From 9e027888bb59f6fcf4c373508e5ee96bbd7af873 Mon Sep 17 00:00:00 2001
From: Luc Schrijvers <begasus@gmail.com>
Date: Mon, 18 Aug 2025 16:38:11 +0200
Subject: Reset qt_policy


diff --git a/CMakeLists.txt b/CMakeLists.txt
index 21abdae..e2d81f6 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -153,7 +153,7 @@ if (ANDROID)
     include(ECMAddAndroidApk)
 endif ()
 
-qt_policy(SET QTP0004 NEW)
+qt_policy(SET QTP0004 OLD)
 
 ecm_setup_version(${PROJECT_VERSION}
     VARIABLE_PREFIX TOKODON
-- 
2.50.1

